### DISCUSSION TITLE
Feature Request - Add css class option to add_option method

### DISCUSSION THREAD

**Discussion by vikasbhvsr** [jeffreyvr/wp-settings#8] [Ideas]
First of all, a huge thanks for creating this amazing package. It is tremendously helpful to build quick WP Settings page with simple API and needless to say, it saves tons of time.

I have a feature request to add `css_class` option to the `add_option` method so we can pass built-in classes like `regular-text`, `large-text`, `all-options` and so on or custom css class to `input` elements. I suggest that the option would have capability to more than one css class separate by space.

I sincerely you consider this.

Thank you.

**Comment by jeffreyvr** [comment]
Feature implemented in 1.1.0:

```php
$section->add_option('color', [
    'name' => 'option_1',
    'label' => __('Option 1', 'textdomain'),
    'css' => [
        'input_class' => 'example-class',
        'label_class' => 'example-class',
    ]
] );
```

Thanks for suggesting!
